langchain/callbacks/tracers/langchain.py

```diff
@@ -90,13 +101,80 @@ def load_default_session(self) -> TracerSession:
         return self._load_session("default")
 
 
-class LangChainTracerV2(LangChainTracer):
+def _get_tenant_id() -> Optional[str]:
+    """Get the tenant ID for the LangChain API."""
+    tenant_id: Optional[str] = os.getenv("LANGCHAIN_TENANT_ID")
+    if tenant_id:
+        return tenant_id
+    endpoint = _get_endpoint()
+    headers = _get_headers()
+    response = requests.get(endpoint + "/tenants", headers=headers)
+    response.raise_for_status()
+    tenants: List[Dict[str, Any]] = response.json()
+    if not tenants:
+        raise ValueError(f"No tenants found for URL {endpoint}")
+    return tenants[0]["id"]
+
+
+class LangChainTracerV2(BaseTracer):
     """An implementation of the SharedTracer that POSTS to the langchain endpoint."""
 
-    @staticmethod
-    def _convert_run(run: Union[LLMRun, ChainRun, ToolRun]) -> Run:
-        """Convert a run to a Run."""
+    def __init__(self, **kwargs: Any) -> None:
+        """Initialize the LangChain tracer."""
+        super().__init__(**kwargs)
+        self._endpoint = _get_endpoint()
+        self._headers = _get_headers()
+        self.tenant_id = _get_tenant_id()
+
+    def _get_session_create(
+        self, name: Optional[str] = None, **kwargs: Any
+    ) -> TracerSessionBase:
+        return TracerSessionV2Base(name=name, extra=kwargs, tenant_id=self.tenant_id)
```

Comment:
Why is this `TracerSessionV2Base` and not `TracerSessionV2Create`?

Reply:
Didn't seem necessary to make a `Create` class before it required unique variables from the `Base` class, but I can mirror the structure of the v1 sessions